easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-goolf-1.7.20.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'BCFtools'
+version = '1.3.1'
+
+homepage = 'http://www.htslib.org/'
+description = """BCFtools is a set of utilities that manipulate variant calls in the 
+ Variant Call Format (VCF) and its binary counterpart BCF"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/samtools/bcftools/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+dependencies = [
+	('zlib', '1.2.8'),
+	('GSL', '1.16'),
+]
+
+buildopts = " USE_GPL=1 GSL_LIBS='-lgsl -lopenblas'"
```

Comment:
@pescobar use `$LIBBLAS`, there's a reason `-lgfortran` is there (although it's only relevant when compiling Fortran, obviously)

It should be harmless when compiling non-Fortran code.